src/Model/Anime/MusicVideoListItem.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Jikan\Model\Anime;
+
+use Jikan\Model\Common\MusicMeta;
+use Jikan\Model\Common\YoutubeMeta;
+use Jikan\Parser\Anime\MusicVideoListItemParser;
+
+/**
+ * Class MusicVideoListItem
+ *
+ * @package Jikan\Model\Anime
+ */
+class MusicVideoListItem
+{
+    /**
+     * @var string
+     */
+    public string $title;
+
+    /**
+     * @var YoutubeMeta
+     */
+    public YoutubeMeta $video;
+
+    /**
+     * @var MusicMeta
+     */
+    public MusicMeta $meta;
```

Comment:
These should be private if you have getter functions for them according to some paradigm I know but can't name atm. Well we already do it this way in other classes, but I just wanted to highlight that this is not a good practice in general.

Reply:
Good catch